student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(_clasroom, age, name = 'unknown', parent_permission: true)
+    super(age, name, parent_permission)
+    @classroom = classroom
```

Comment:
- [ ] I think there is a syntax error in your `initialize` method parameters. I think you meant `classroom`, not `_clasroom`. Kindly fix this.